src/cli.mjs

```diff
@@ -2836,87 +1537,87 @@ function commonInitPrintFoundContracts() {
-        log.write( cc.sunny( "TokenManagerERC20" ) + cc.debug( "...............address is....." ) +
-            oct( imaState.joTokenManagerERC20Target ) + "\n" );
-        log.write( cc.sunny( "TokenManagerERC721" ) + cc.debug( "..............address is....." ) +
-            oct( imaState.joTokenManagerERC721Target ) + "\n" );
-        log.write( cc.sunny( "TokenManagerERC1155" ) + cc.debug( ".............address is....." ) +
-            oct( imaState.joTokenManagerERC1155Target ) + "\n" );
-        log.write( cc.sunny( "TokenManagerERC721WithMetadata" ) + cc.debug( "..address is....." ) +
-            oct( imaState.joTokenManagerERC721WithMetadataTarget ) + "\n" );
-        log.write( cc.sunny( "CommunityLocker" ) + cc.debug( ".................address is....." ) +
-            oct( imaState.joCommunityLockerTarget ) + "\n" );
-        log.write( cc.sunny( "MessageProxy" ) + cc.debug( "....................address is....." ) +
-            oct( imaState.joMessageProxySChainTarget ) + "\n" );
-        log.write( cc.sunny( "TokenManagerLinker" ) + cc.debug( "..............address is....." ) +
-            oct( imaState.joTokenManagerLinkerTarget ) + "\n" );
-        log.write( cc.sunny( "ERC20" ) + cc.debug( " ..........................address is....." ) +
-            oct( imaState.joEthErc20Target ) + "\n" );
-
-        log.write( cc.bright( "Skale Manager contracts:" ) + "\n" );
-        log.write( cc.sunny( "ConstantsHolder" ) + cc.debug( ".................address is....." ) +
-            oct( imaState.joConstantsHolder ) + "\n" );
-        log.write( cc.sunny( "Nodes" ) + cc.debug( "...........................address is....." ) +
-            oct( imaState.joNodes ) + "\n" );
-        log.write( cc.sunny( "KeyStorage" ) + cc.debug( "......................address is....." ) +
-            oct( imaState.joKeyStorage ) + "\n" );
-        log.write( cc.sunny( "Schains" ) + cc.debug( ".........................address is....." ) +
-            oct( imaState.joSChains ) + "\n" );
-        log.write( cc.sunny( "SchainsInternal" ) + cc.debug( ".................address is....." ) +
-            oct( imaState.joSChainsInternal ) + "\n" );
-        log.write( cc.sunny( "SkaleDKG" ) + cc.debug( "........................address is....." ) +
-            oct( imaState.joSkaleDKG ) + "\n" );
-        log.write( cc.sunny( "SkaleManager" ) + cc.debug( "....................address is....." ) +
-            oct( imaState.joSkaleManager ) + "\n" );
-        log.write( cc.sunny( "SkaleToken" ) + cc.debug( "......................address is....." ) +
-            oct( imaState.joSkaleToken ) + "\n" );
-        log.write( cc.sunny( "ValidatorService" ) + cc.debug( "................address is....." ) +
-            oct( imaState.joValidatorService ) + "\n" );
-        log.write( cc.sunny( "Wallets" ) + cc.debug( ".........................address is....." ) +
-            oct( imaState.joWallets ) + "\n" );
+        log.debug( "IMA contracts(Main Net):" );
```

Comment:
is it possible to print all addresses in loop?

Reply:
everything possible is printed at startup, excluding key names and URLs